### ISSUE TITLE
Header requires spaces where is shouldn't

### ISSUE THREAD

**Issue by faldor20** [roc-lang/roc#6322]
This code fails to parse.
```elm
interface Test
  exposes[]
  imports[]
```
Adding spaces after exposes and imports fixes it.
 ```elm
interface Test
  exposes []
  imports []
```
This should probably parse fine, but be fixed by the formatter. 
I imagine this may not matter after the rework of module headers in #6201 , but I thought I'd log it anyway

**Comment by imclerran** [comment]
This issue can probably be closed since PR #6683...

**Comment by Anton-4** [comment]
Thanks for bringing it up @imclerran!
